vcpkg/get_started/get-started.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Install and use packages with vcpkg
+description: Tutorial guides the user through the process of installing and using packages with vcpkg.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages with vcpkg
+
+This tutorial shows you how to create a C++ "Hello World" program that uses the `fmt` library with CMake and vcpkg. You'll install dependencies, configure, build, and run a simple application.
+
+## Prerequisites
+
+- A terminal to work in.
+- A C++ compiler
+- [CMake](https://cmake.org/download/)
+- [Git](https://git-scm.com/downloads)
+
+[!INCLUDE [setup-vcpkg](setup-vcpkg.md)]
+
+
+## 2 - Set up the project
+
+1. Create the project directory
+
+    This tutorial assumes that your project will be located in a directory named `helloworld`. The `helloworld` directory should be a sibling (or peer) of your vcpkg directory, meaning both should reside under the same parent directory. If you intend to place your project somewhere else, replace any references to that path with your chosen path.
```

Comment:
Maybe we can just say "Set `VCPKG_ROOT` to your vcpkg directory"? They eventually have to do that anyway.